freqtrade/tests/optimize/test_backtesting.py

```diff
@@ -49,30 +49,31 @@ def test_backtest_1min_ticker_interval(default_conf, mocker):
     assert not results.empty
 
 
-def trim_dataframe(df, num):
-    new = dict()
-    for pair, pair_data in df.items():
-        new[pair] = pair_data[-num:]  # last 50 rows
+def trim_dictlist(dl, num):
+    new = {}
+    for pair, pair_data in dl.items():
+        # Can't figure out why -num wont work
+        #new[pair] = pair_data[-num:]
+        new[pair] = pair_data[-100:]
     return new
 
 
 def load_data_test(what):
     data = optimize.load_data(ticker_interval=1, pairs=['BTC_UNITEST'])
-    data = trim_dataframe(data, -40)
+    data = trim_dictlist(data, -100)
     pair = data['BTC_UNITEST']
-
     # Depending on the what parameter we now adjust the
-    # loaded data:
+    # loaded data looks:
     # pair :: [{'O': 0.123, 'H': 0.123, 'L': 0.123,
     #           'C': 0.123, 'V': 123.123,
     #           'T': '2017-11-04T23:02:00', 'BV': 0.123}]
+    o = 0.001
+    h = o
+    ll = o
+    c = o
+    ll -= 0.0001
+    h += 0.0001
```

Comment:
I think it would be more readable to use `open`, `close`, `high`, `low` as variable names instead of those single characters.